test/e2e/reachability.go

```diff
@@ -55,10 +55,28 @@ func (pod Pod) PodName() string {
 	return podName
 }
 
+type PodConnectivityMark uint8
+
+const (
+	Connected     PodConnectivityMark = 0
+	UnknownStatus PodConnectivityMark = 1
```

Comment:
Maybe `Error` would be a better name than `UnknownStatus`. You only use it in case of an error.

Reply:
Changed.